lib/core/test/integration/cardano-wallet-core-integration.cabal

```diff
@@ -0,0 +1,68 @@
+name:                cardano-wallet-core-integration
```

Comment:
Let's not make it a library, but have it simply located in `test/core` and referred to in the corresponding integration tests using `hs-source-dirs`. Having test code as a library will have further complications.

Reply:
good idea, will push changes in a while